tests/integration/usage_statistics/test_usage_statistics_messages.py

```diff
@@ -1299,6 +1299,36 @@ def test_generate_messages_with_defaults():
             "x-forwarded-for": "00.000.00.000, 00.000.000.000",
         },
     ],
+    "profiler.result.get_expectation_suite": [
+        {
+            "event": "profiler.result.get_expectation_suite",
+            "event_payload": {
```

Comment:
just want to make sure this is representative of _all_ possible payload values. it's not required in order to infer the schema, but for my sake i think i'd like to have the examples be as such.

Reply:
@Super-Tanner Confirmed -- yes, and the new test messages verify that.  Thanks.